android/app/src/main/java/edu/gatech/cs/environmentalodors/MapsActivity.java

```diff
@@ -247,6 +248,7 @@ public void generateFakeData() {
         LatLng center = DEFAULT_LOCATION; // approximately atlanta
         float radius = 0.2f;
         int reportCount = 5;
+        int commentCount = 5;
```

Comment:
Better practice to declare variables as close as possible to their location of use.